Travis lint (#2)
EGI-Foundation/datahub-documentation: source/clients.rst

Clients
-------

> The `Oneclient` code and basic documentation is available on GitHub:

Reviewer: surely `oneclient` should be written lower-case if it's in code blocks, and upper-case if it's the name of the tool?

Author: Yes the case usage you mention is appropriate

Testing Oneclient in a Oneclient docker container with NFS or samba
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Docker containers for the Oneclient are available, the existing versions can be
seen in docker hub: https://hub.docker.com/r/onedata/oneclient/tags

It's possible to use the most recent version by specifying the `latest` tag.

> We also recommend to use the same version that can be seen on the Onezone and

Reviewer: ```suggestion
We also recommend using same version as shown on the Onezone and
```

Author: Implemented
